R/save.r

```diff
@@ -117,7 +117,10 @@ plot_dim <- function(dim = c(NA, NA), scale = 1, units = c("in", "cm", "mm"),
   dim
 }
 
-plot_dev <- function(device, filename, dpi = 300) {
+plot_dev <- function(device, filename = NULL, dpi = 300) {
```

Comment:
For my own education, could you explain why we now need `filename = NULL` here but in some other cases (e.g., `make_summary_fun()`) we don't need to add a default?

Reply:
The default doesn't make sense there to me, the function needs arguments or fails. But e.g. `manual_scale()` could use a better error message.